pkg/daemon/init_linux.go

```diff
@@ -90,15 +90,43 @@ func changeProvideNicName(current, target string) (bool, error) {
 		klog.Errorf("failed to set link %s up: %v", target, err)
 		return false, err
 	}
+	klog.Infof("link %s has been renamed as %s", current, target)
+
+	// wait systemd-networkd to finish interface configuration
```

Comment:
Does 50 millisecond enough for networkd to configure the interface? And this part should be moved to a separated function.

Reply:
`systemd-networkd` monitors netlink events and responds quickly. 50ms is enough in VM tests.